utils/rpms/daos.spec

```
@@ -562,6 +562,9 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 # No files in a shim package
 
 %changelog
+* Wed Apr  6 2022 Joseph Moore <joseph.moore@intel.com> 2.1.100-22
+- ADD build depends entries for UCX libraries.
```

If you happen to need to update this PR for anything else:

```suggestion
- Add build depends entries for UCX libraries.
```